app/db/models/_tests_/user.test.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-undef */
+import userFactory from '../../factories/user.js'
+const { User } = require('../../models')
+
+describe('creates a user', () => {
+  let user
+
+  // Before any tests run, clear the DB and run migrations with Sequelize sync()
+  // note: we may not need the pretest hooks if we do it this way?
+  beforeAll(async () => {
+    await User.sequelize.sync({ force: true })
+  })
+
+  beforeEach(async () => {
+    user = await userFactory()
+  })
+  it('created user to match', async () => {
+    user = await userFactory({ id: 'thefirstuser' })
+    expect(user.id).toBe('thefirstuser')
+  })
+  it('has a default role of USER', async () => {
+    expect(user.roles[0]).toBe('USER')
+  })
+  it('adds admin to role', async () => {
+    user.roles.push('ADMIN')
+    expect(user.roles[1]).toBe('ADMIN')
+  })
+  it('adds admin to role', async () => {
+    user.roles.push('SPACE')
+    // should fail validation, but im not certain how to check this
```

Comment:
so this should _not_ work, but I think Sequelize won't trigger a validation error until the model is updated or saved.